tools/firmware-utils/src/tplink-safeloader.c

```diff
@@ -77,6 +77,7 @@ struct device_info {
 	const char *support_list;
 	char support_trail;
 	const char *soft_ver;
+	uint32_t soft_ver_compat_level;
```

Comment:
The compat_level-changing upgrade described in the commit message (EAP225-Outdoor FWv1.4.1 to FWv1.7.0) was necessary to enable flashing of OpenWrt with checksum disabling (v1.4.1 didn't have the required `cliclientd` utility). 

Since all firmwares I know that have the `cliclientd` utility also appear to preserve the original compatibility level, it *might* be possible to just use 0xfffffffe for OpenWrt and call it a day. However, I don't feel very confident about this and would rather stay close to TP-Link stock images with a low, but device-specific compat_level.